test/parallel/test-vm-break-on-sigint.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+const common = require('../common');
+const { Worker, isMainThread } = require('worker_threads');
+const vm = require('vm')
+
+if (isMainThread) {
+    for (let i = 0; i < 10; i++) {
+      const worker = new Worker(__filename);
+      worker.on('exit', common.mustCall());
+    }
+} else {
+    const ctx = vm.createContext({});
+    for (let i = 0; i < 10000; i++) {
+        vm.runInContext('console.log(1)', ctx, { breakOnSigint: true });
+    }
+}
```

Comment:
This won't work as expected when run from a worker context (`tools/test.py --worker test/parallel/test-vm-break-on-sigint`).
Also, a line return at the end of the file would be nice:
```suggestion
}

```

Reply:
Thanks for review. I have fix the code style but i do not understand what this means `This won't work as expected when run from a worker context (tools/test.py --worker test/parallel/test-vm-break-on-sigint)`. I run `tools/test.py --worker test/parallel/test-vm-break-on-sigint` and see `[00:01|% 100|+   1|-   0]: Done`.

Reply:
> This won't work as expected when run from a worker context (`tools/test.py --worker test/parallel/test-vm-break-on-sigint`). Also, a line return at the end of the file would be nice:

Thanks for review. I have fix the code style but i do not understand what this means `This won't work as expected when run from a worker context (tools/test.py --worker test/parallel/test-vm-break-on-sigint)`. I run `tools/test.py --worker test/parallel/test-vm-break-on-sigint` and see `[00:01|% 100|+   1|-   0]: Done`.

Reply:
When run on worker context, `isMainThread` is false, so it's only going to create a 1000 vm context and exit successfully. But it won't have created the 10 sub-`Worker`s and ensure they exit. Maybe that's OK fro this test though, I don't know, as long as the test is not crashing it's not a blocking concern.

Reply:
We normally set and check an environment variable instead of using `isMainThread` to run the test from a worker instance itself.

Reply:
Thanks ! I see something like `process.env.HAS_STARTED_WORKER` in other test file. I will fix it later.

Reply:
FWIW, there are other tests in `test/parallel` that use `worker_threads.isMainThread`, like https://github.com/nodejs/node/blob/ed7631dae984c9b5a7ef722119dc9363af7472a5/test/parallel/test-crypto-worker-thread.js#L12 for example. We should probably find a nicer way to make sure that such uses don't slip through.